tests/upgrades/test_repository.py

```diff
@@ -143,29 +136,24 @@ def test_pre_scenario_custom_repo_check(self, target_sat, save_test_data):
         lce = target_sat.api.LifecycleEnvironment(organization=org).create()
 
         product = target_sat.api.Product(organization=org).create()
-        target_sat.create_custom_html_repo(rpm1)
-        repo = target_sat.api.Repository(
-            product=product.id, url=f'{target_sat.url}/pub/custom_repo'
-        ).create()
+        repo = target_sat.api.Repository(product=product.id, url=settings.repos.yum_1.url).create()
         repo.sync()
-
-        content_view = publish_content_view(org=org, repolist=repo)
+        content_view = target_sat.api.ContentView(organization=org).create()
+        content_view.repository = repo if type(repo) is list else [repo]
+        content_view = content_view.update(['repository'])
+        content_view.publish()
+        content_view = content_view.read()
         content_view.version[0].promote(data={'environment_ids': lce.id})
-
-        subscription = target_sat.api.Subscription(organization=org).search(
-            query={'search': f'name={product.name}'}
-        )[0]
         ak = target_sat.api.ActivationKey(
             content_view=content_view, organization=org.id, environment=lce
         ).create()
-        ak.add_subscriptions(data={'subscription_id': subscription.id})
```

Comment:
can we update this to add subscriptions only if the org has SCA enabled instead?

Reply:
@Gauravtalreja1 did you mean "only if the org has SCA disabled?" Adding subscriptions when SCA is enabled should result in an error.